LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Splunk Inc.
+   Copyright (c) 2017-2022 Splunk Inc.
```

Comment:
Just wanna share that going forward you only need to update the license year in the App json https://github.com/splunk-soar-connectors/parser/blob/858e535ab1e367def800545988075c3405dd4e3a/parser.json#L14

Other license info will be automatically updated by the `start-release` action

Reply:
Thank you.